beprof/profile.py

```diff
@@ -10,8 +10,7 @@
 
 class Profile(curve.Curve):
     """
-    General profile characterized by rising and falling edge
-    Might be depth profile (along Z axis) or lateral profile (X or Y scan)
+    General profile characterized by rising and falling edge (X or Y scan)
```

Comment:
Remove stuff in brackets. Lets keep things general. Depth profile (Z scan) is also 2D